Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3483,7 +3501,7 @@ def panorama_list_rules(xpath: str, name: str = None, filters: dict = None, quer
 
     if query:
         params["xpath"] = f'{params["xpath"]}[{query}]'
-    elif xpath_filter := build_xpath_filter(name, filters):
+    elif xpath_filter := build_xpath_filter(name_match=name, filters= filters):
```

Comment:
```suggestion
    elif xpath_filter := build_xpath_filter(name_match=name, filters=filters):
```

should be without space, my bad from last review.